server/tests/document.test.js

```diff
@@ -0,0 +1,47 @@
+const chaiHttp = require('chai-http');
+const chai = require('chai');
+const supertest = require('supertest');
+const should = require('chai').should;
+expect = require('chai').expect;
+chai.use(chaiHttp);
+
+api = supertest('http://localhost:8000');
+
+describe('Document', () => {
+    it('should create a document on /document/POST', (done) => {
+        api.get('/api/document/', (error, response, body) => {
```

Comment:
Why are we doing get when testing post and also expecting a 200 and not 201(Created)?